.github/workflows/create-next-milestone.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     steps:
       - name: Get next minor version
         id: semvers
-        uses: WyriHaximus/github-action-next-semvers@0.1.0
+        uses: WyriHaximus/github-action-next-semvers@b135abb108d66990a85e18623d906404f4350ce4
         with:
           version: ${{ github.event.milestone.title }}
       - name: Create next milestone
-        uses: WyriHaximus/github-action-create-milestone@0.1.0
+        uses: WyriHaximus/github-action-create-milestone@ab85332e3150ec018daf497a0f761fe69d52bc7d
```

Comment:
Should we also upgrade this one? ab85332e3150ec018daf497a0f761fe69d52bc7d is actually for the old 0.1.0 release (the same one we had), not [the latest v1.0.0 release](https://github.com/WyriHaximus/github-action-create-milestone/tags).

Reply:
Same as comment above.